src/components/app/ListOfPublishedProfiles.js

```diff
@@ -283,19 +283,33 @@ type State = {|
 |};
 
 export class ListOfPublishedProfiles extends PureComponent<Props, State> {
+  _isMounted = false;
+
   state = {
     profileDataList: null,
   };
 
   async componentDidMount() {
+    this._isMounted = true;
     const profileDataList = await listAllProfileData();
```

Comment:
You need to remplace this line with `this.loadAllProfileData()`, I believe that's your problem for the tests (and probably that's a real bug).